pmlc/rt/vulkan/tests/BUILD

```diff
@@ -2,8 +2,26 @@
 
 package(default_visibility = ["//visibility:public"])
 
-load("//pmlc:lit.bzl", "glob_lit_tests")
+load("//pmlc:lit.bzl", "glob_lit_tests", "lit_test")
+load("//bzl:plaidml.bzl", "plaidml_cc_test")
 
 glob_lit_tests(
     data = ["//pmlc/rt/vulkan:vkprobe"],
 )
+
+plaidml_cc_test(
+    name = "cc_test_gpu",
+    srcs = ["edsl_test_gpu.cc"],
+    tags = ["manual"],  # the `edsl_test.cc.test` rule will run this test via lit
+    deps = [
+        "//plaidml:testenv",
+        "//plaidml/edsl",
+        "//plaidml/exec",
+        "//pmlc/rt/vulkan:link",
+    ],
+)
+
+lit_test(
+    name = "edsl_test_gpu.cc",
+    data = [":cc_test_gpu"],
+)
```

Comment:
I was hoping we could put this in the other directory because now we need a special `lit.local.cfg` which uses `vkprobe` to enable/disable this specific test.

Reply:
I see, will use vkprobe as a trigger.